web-app/src/components/YugiohCard/YugiohCard.js

```diff
@@ -138,8 +138,9 @@ function YugiohCard({ height, notFull, player, row, zone, cardName, modal, isHer
       };
    }, []);
 
-   const margin = !notFull && (height - height / CARD_RATIO) / 2 + 2;
+   const margin = !notFull && (height - height / CARD_RATIO) / 2 + 2; // the +2 is to leave a little space between cards

```

Comment:
I think you're not flooring here when you want to be?

```suggestion
   const width = Math.floor(height / CARD_RATIO),
   const margin = !notFull && (height - width) / 2 + 2; // the +2 is to leave a little space between cards
```

And then replace `width: Math.floor(height / CARD_RATIO),` with just `width`.

Reply:
This does make more sense. I'll do it this way.